src/liballoc/tests/lib.rs

```diff
@@ -25,6 +25,7 @@
 #![feature(unboxed_closures)]
 #![feature(exact_chunks)]
 #![feature(repeat_generic_slice)]
+#![feature(trim_direction)]
```

Comment:
This can be removed here and in the doc tests if we are landing these as stable.

Reply:
Whoops, fixed.

Reply:
I don't think the fix has been pushed yet.

Reply:
I just noticed I pushed the wrong branch. It's definitely fixed now 😅